generator/src/test/java/com/scottlogic/deg/generator/cucumber/utils/CucumberTestState.java

```diff
@@ -26,6 +26,8 @@
      */
     public Boolean shouldViolate = false;
 
+    public Boolean overwriteOutputFiles = true;
```

Comment:
We dont need to be able to vary this in cucumber, can we pass `true` in to wherever needs it directly rather than via a field

Reply:
okay, removed the field